Source/src/core/Scene.cpp

```diff
@@ -256,6 +257,12 @@ void Hachiko::Scene::Load(const YAML::Node& node, bool meshes_only)
         }
         // Pass skybox with used uids to be loaded
         skybox = new Skybox(cube);
+
+        if (node[IBL].IsDefined() && node[IBL].as<bool>())
```

Comment:
It may be a good idea to store if ibl is enabled or not and load it like `node[IBL].as<bool>()` instead.

Reply:
Isn't that what we are doing? Do you mean to always build it but enabling it or disabling depending on the setting?